source/Octopus.Tentacle/Communications/AssemblyMappingSerializationBinderDecorator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Newtonsoft.Json.Serialization;
+
+namespace Octopus.Tentacle.Communications
+{
+    public class AssemblyMappingSerializationBinderDecorator : ISerializationBinder
```

Comment:
This is important - maps from the old assembly name (Octopus.Shared) to the new assembly name (Octopus.Tentacle.Contracts).